bool gimbal_finish_left_nine
bool gimbal_finish_right_nine
bool deduce_blood
bool finish_follow
bool finish_climbing
